src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8701,6 +8706,10 @@ def astype(
                 labels, include_index=False
             )
         )
+        data_column_snowpark_pandas_types = [
```

Comment:
we also need to make sure to convert the snowflake type to int.

Reply:
Yep will do.